upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -639,6 +639,91 @@ There you can find links to upgrade notes for other versions too.
         ```
         - for more information you can see the [PR](https://github.com/shopsys/shopsys/pull/1461)
 
+- add image and iframe LazyLoad ([#1483](https://github.com/shopsys/shopsys/pull/1483))
+
+    - update [listGrid.html.twig](https://github.com/shopsys/project-base/blob/master/packages/framework/src/Resources/views/Admin/Content/Advert/listGrid.html.twig)
+        ```diff
+          {% if row.advert.type == TYPE_IMAGE %}
+        -    {{ image(row.advert, {size: 'original', height: 30}) }}
+        +    {{ image(row.advert, {size: 'original', height: 30, lazy: false}) }}
+           {% else %}
+        ```
+    - update [imageuploadFields.html.twig](https://github.com/shopsys/project-base/blob/master/packages/framework/src/Resources/views/Admin/Form/imageuploadFields.html.twig)
+        ```diff
+          <div class="list-images__item__image js-image-upload-preview {% if isRemoved %}list-images__item__in--removed{% endif %}">
+        -     {{ image(image, {size: 'original', height: '100', type: image_type}) }}
+        +     {{ image(image, {size: 'original', height: '100', type: image_type, lazy: false}) }}
+          </div>
+          <div class="form-line__item">
+        -      {{ image(entity, { size: 'original', height: 100, type: image_type }) }}
+        +      {{ image(entity, { size: 'original', height: 100, type: image_type, lazy: false }) }}
+          </div>
+        ```
+
+    - update [ajaxMoreLoader.js](https://github.com/shopsys/project-base/blob/master/project-base/src/Shopsys/ShopBundle/Resources/scripts/frontend/components/ajaxMoreLoader.js)
+        ```diff
+                 $paginationToItemSpan.text(paginationToItem);
+                 updateLoadMoreButton();
+        +        Shopsys.lazyLoadCall.inContainer($currentList);
+                 Shopsys.register.registerNewContent($nextItems);
+             }
+        ```
+    - update [productList.AjaxFilter.js](https://github.com/shopsys/project-base/blob/master/project-base/src/Shopsys/ShopBundle/Resources/scripts/frontend/product/productList.AjaxFilter.js)
+        ```diff
+                 $productsWithControls.show();
+        +        Shopsys.lazyLoadCall.inContainer($productsWithControls);
+                 Shopsys.register.registerNewContent($productsWithControls);
+             };
+        ```
+    - add new file [project-base/src/Shopsys/ShopBundle/Resources/scripts/frontend/lazyLoadInit.js](https://github.com/shopsys/shopsys/blob/master/project-base/src/Shopsys/ShopBundle/Resources/scripts/frontend/lazyLoadInit.js)
```

Comment:
almost there, but this should be without project-base/ in path because people are using project-base so their paths starts without it...

and please add full path for all files you have updated / added ...